build-system/tasks/runtime-test/status-report.js

```diff
@@ -113,4 +113,12 @@ exports.reportAllExpectedTests = async buildTargets => {
       await postReport(`${type}/${subType}`, action);
     }
   }
+}
+
+module.exports = {
+  reportTestErrored,
+  reportAllExpectedTests,
```

Comment:
Care to export these alphabetically if you're already here? 😅

Reply:
Done.